flink-table/flink-table-planner-blink/src/main/resources/META-INF/NOTICE

```diff
@@ -12,9 +12,9 @@ This project bundles the following dependencies under the Apache Software Licens
 - com.fasterxml.jackson.core:jackson-databind:2.10.1
 - com.jayway.jsonpath:json-path:2.4.0
```

Comment:
Not sure but if we can't remove `net.minidev:accessors-smart` and `org.ow2.asm` we should also add them here.

Reply:
Yes, thanks for the reminder.